warmup.krun

```diff
@@ -6,7 +6,7 @@ from krun.vm_defs import (PythonVMDef, LuaVMDef, JavaVMDef, GraalVMDef,
 from krun import EntryPoint
 
 # Who to mail
-MAIL_TO = ["vext01@gmail.com", "mount.sarah@gmail.com"]
+MAIL_TO = []
 
 # Maximum number of error emails to send per-run
 #MAX_MAILS = 2
```

Comment:
Ah, it's just there to show you can change it if you like. We could just remove it tbh.